packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-line-settings.tsx

```diff
@@ -73,13 +74,34 @@ const LockedLineSettings = (props: Props) => {
 
     // Check if the line has length 0.
     const isInvalid = kvector.equal(point1.coord, point2.coord);
-
-    function getPrepopulatedAriaLabel() {
+    /**
+     * Generate a prepopulated aria label for the line, with the math
+     * details converted into spoken words.
+     */
+    async function getPrepopulatedAriaLabel() {
         let visiblelabel = "";
+        let point1VisibleLabel = "";
+        let point2VisibleLabel = "";
```

Comment:
Why are these variables called `*VisibleLabel` if this function is generating an aria label?

...oh, I think I get it. The term "label" is overloaded. We're using "visible label" to contrast with "aria label". If we just called the point labels "labels" that might be confusing.

Reply:
Yeah, it's expressing the the locked (visible) label that's associated with this figure is being added to the aria label.